README.md

```diff
@@ -5,6 +5,21 @@ Demonstrates Iteria React Lowcode features:
 * [Translation sheet](https://www.youtube.com/watch?v=5-U2vfgHkMA&t=396s) - spreadsheet with all messages and their translations
 
 You can run this demo locally, [edit in StackBlitz](https://stackblitz.com/github/iteria-app/example-material-ui) or using Gitpod. 
```

Comment:
Inak StackBlitz nam stale funguje? Mozno by mohol fungovat aj codesandbox.

Reply:
StackBlitz funguje, otestujem codesandbox

Reply:
StackBlitz má problém s niektorými dependencies a `vite` dev mode, nedokáže nájsť niektoré bundles a `codesandbox` mi nedovolí otvoriť `example-mui`, myslí si že je to private repo